src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -462,12 +462,11 @@ public void incomingPacketInfo(
         if (logger.isTraceEnabled())
         {
             logger.trace(diagnosticContext
-                .makeTimeSeriesPoint("incoming_packet", nowMs)
+                .makeTimeSeriesPoint("in_pkt", nowMs)
                 .addKey("rbe", hashCode())
```

Comment:
for example, this as is is tough...i assumed it's 'this is the hash of this remote bandwidth estimator instance' ? so maybe `rbe_id` would be better?